modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1805,7 +1805,7 @@ up    <string name="player_sleep_in_one_chapter">Sleeping in 1 chapter</string>
     <string name="onboarding_recommendations_make_pocket_casts_yours">Make Pocket Casts yours by selecting your favorite podcasts or import them from another podcast player. You can always do this later.</string>
     <string name="onboarding_recommendations_import">Import</string>
     <string name="onboarding_recommendations_more">More %s</string>
-    <string name="onboarding_upgrade_everything_you_love_about_pocket_casts_plus">Everything you love about Pocket&#160;Casts, plus more</string>
+    <string name="onboarding_upgrade_everything_you_love_about_pocket_casts_plus">Unlock exclusive features with Pocket Casts Plus</string>
```

Comment:
I was wondering if we should use this string to avoid app name wrapping to next line:

`Unlock exclusive features with Pocket&#160;Casts Plus`

** I understand it is not used this way in other places but it caught my attention.

<img width=320 src="https://github.com/user-attachments/assets/6008fa12-9568-4dbb-8a6f-9239a3bf2013"/>

Reply:
Yeah, makes to not break `Pocket Casts`. Good catch!